### ISSUE TITLE
Requesting Updates to tutorial for Azure Data Explorer

### ISSUE THREAD

**Issue by jaimeatwork** [MicrosoftDocs/dataexplorer-docs#817]
Hello, Thanks for making this tutorial

The section `Create the data connection for diagnostic metrics and logs and activity logs` looks like its referencing an old version of the console, compression settings are not referenced in the instructions.

Additionally, I followed the steps several times, double checking my work along the way and was not able to get events to populate in my table. Did something change that's not obvious breaking these instructions? Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9fe49562-852d-db87-a190-8e6f267951fc
* Version Independent ID: 08c75879-50de-cf95-ecac-226e93285a66
* Content: [Tutorial: Ingest monitoring data in Azure Data Explorer without code](https://docs.microsoft.com/en-us/azure/data-explorer/ingest-data-no-code?tabs=activity-logs)
* Content Source: [data-explorer/ingest-data-no-code.md](https://github.com/MicrosoftDocs/dataexplorer-docs/blob/master/data-explorer/ingest-data-no-code.md)
* Service: **data-explorer**
* GitHub Login: @orspod
* Microsoft Alias: **orspodek**

**Comment by BhargaviAnnadevara-MSFT** [comment]
@jaimeatwork Thanks for the feedback. We're investigating this and will get back to you shortly.

**Comment by ohadbitt** [comment]
Hello
Its OK that the section is not updated as it should work with default values (its not compressed)- but we will update it.
in the section
'See data flowing to your event hubs'
do you see trafic to EH ? if yes - do you see trafic from EH?

If not - are you sure the categories you selected produce data ?

**Comment by jaimeatwork** [comment]
Howdy, yes, it looks like it took a while for the data to populate (maybe my source account isn't active enough or it takes longer to populate than I expected) and since I created and deleted the data connection a few times, it may have resulted in lost data, I'm not certain. 

After a weekend of letting it run, I can back, was able to simulate some activity like creating and deleting a virtual machine, and I am seeing activity like I would expect.  In Azure Active Directory, these are the audit log options I have selected

![image](https://user-images.githubusercontent.com/84590541/133106996-01da6e69-8dcd-4faf-95bc-5d4f1322bae5.png)

I am seeing events flow through the event hub now 

![image](https://user-images.githubusercontent.com/84590541/133111214-a85ac143-547e-4796-bdaf-ee3004f495f8.png)

It looks like a lot of this was a misunderstanding on my part and not being sure what to expect when actions were completed and results didn't soon after match what I saw in the documentation. Thanks for your time.
